moped-editor/src/views/staff/StaffListViewQueryConf.js

```diff
@@ -140,7 +141,9 @@ export const StaffListViewQueryConf = {
     },
   },
   order_by: {},
-  where: {},
+  where: {
+    status_id: "_eq: 1"
```

Comment:
if we are going to only get the active users, do we still need to show the column `status_id`?

Reply:
We don't, good call! 👍

Reply:
I agree with Chia. Doesn't seem necessary

Reply:
It's deleted now :)